Source/PeterKottas.DotNetCore.WindowsService.Example/ExampleService.cs

```diff
@@ -1,25 +1,25 @@
-﻿using PeterKottas.DotNetCore.WindowsService.Interfaces;
+﻿using Microsoft.Extensions.PlatformAbstractions;
+using PeterKottas.DotNetCore.WindowsService.Interfaces;
 using System;
 using System.IO;
-using System.Diagnostics;
-using Microsoft.Extensions.PlatformAbstractions;
-using System.ServiceProcess;
-using System.Threading.Tasks;
+using System.Timers;
 
 namespace PeterKottas.DotNetCore.WindowsService.Example
 {
-    public class ExampleService : IMicroService
+	public class ExampleService : IMicroService
```

Comment:
It seems your settings for tabs are different to what the rest of the code bases uses, would you pls be able to reformat this to match existing formating?

Reply:
No problem.